tests/gen_testdata.py

```diff
@@ -158,7 +158,7 @@ def compute_one_result(
 
         for a, b, balance in zip(addresses, addresses[1:], balances_before):
             self.contract.functions.setAccount(
-                a, b, 100_000_000, 100_000_000, 0, 0, 0, 0, 0, balance
+                a, b, 100_000_000, 100_000_000, 0, 0, balance
```

Comment:
I guess this should be `a, b, 100_000_000, 100_000_000, 0, 0, False, 0, balance` idk why it was broken before or what it is used for. https://github.com/trustlines-protocol/contracts/blob/master/contracts/tests/TestCurrencyNetwork.sol#L71

Reply:
:heavy_check_mark: